lms/templates/ccx/schedule.html

```diff
@@ -29,22 +29,24 @@
   <div id="new-ccx-schedule"></div>
 </div>
 
-<section id="enter-date-modal" class="modal" aria-hidden="true">
-  <div class="inner-wrapper" role="dialog">
+<section id="enter-date-modal" class="modal"
+         tabindex="-1" role="dialog" aria-labelledby="ccx_schedule_set_date_heading">
+  <div class="inner-wrapper">
     <button class="close-modal">
-      <i class="icon fa fa-remove"></i>
-      <span class="sr">
-        ${_("Close")}
-      </span>
+      <i class="icon fa fa-remove" aria-hidden="true"></i>
+      <span class="sr">${_("Close")}</span>
     </button>
     <header>
-      <h2></h2>
+      <h2 id="ccx_schedule_set_date_heading"></h2>
     </header>
-    <form role="form">
+    <form>
       <div class="field datepair">
-        <label></label>
-        <input placeholder="Date" class="date" type="text" name="date"/ size="11">
-        <input placeholder="Time" class="time" type="text" name="time"/ size="6">
+        <!-- Translators: This explains to people using a screen reader how to interpret the format of YYYY-MM-DD -->
```

Comment:
if this is a Mako file, your syntax is wrong. See http://edx.readthedocs.org/projects/edx-developer-guide/en/latest/internationalization/i18n.html#mako-template-files